pkgs/development/compilers/gofer/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, fetchurl, lib }:
+
+stdenv.mkDerivation rec {
+  pname = "gofer";
+  version = "2.30b";
+
+  enableParallelBuilding = true;
+
+  makeFlags = [ "CFLAGS=-O2" ];
+
+  unpackPhase = ''
+    mkdir -p gofer
+    cd gofer
+    unpackFile $src
+  '';
+
+  patchPhase = ''
+    substituteInPlace prelude.h --replace 'malloc.h' 'stdlib.h'
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mkdir -p $out/lib
+    # cp $src/standard.prelude $out/lib
```

Comment:
```suggestion
```

Reply:
May be needed at run time, need to check.